src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/RestrictedPatientsSearchResourceTest.kt

```diff
@@ -274,6 +274,25 @@ class RestrictedPatientsSearchResourceTest : QueueIntegrationTest() {
     }
   }
 
+  @Nested
+  inner class SupportingPrisonIds {
+    @Test
+    fun `finds restricted patients with a single supporting prison`() {
+      restrictedPatientSearch(
+        RestrictedPatientSearchCriteria(null, null, null, listOf("DNI")),
+        "/results/restrictedPatientsSearch/search_results_supporting_prison_DNI.json"
+      )
+    }
+
+    @Test
+    fun `finds restricted patients with a multiple supporting prisons`() {
+      restrictedPatientSearch(
+        RestrictedPatientSearchCriteria(null, null, null, listOf("DNI", "MDI", "NONE")),
```

Comment:
Just wondering why "NONE" on the request?  What's that testing?

Reply:
Just making sure it ignores ids without matches